health/micro-ui/web/micro-ui-internals/packages/modules/health-hrms/src/hooks/project/usePaginatedProjectSearch.js

```
@@ -0,0 +1,82 @@
+    limit: DEFAULT_LIMIT,
+    offset,
+    includeDescendants: false,
+  }), [tenantId, offset]);
+
+  const { isLoading, data, isError, error } = Digit.Hooks.useCustomAPIHook({
+    url: Urls.hcm.searchProject,
+    params,
+    body: requestBody,
+    config: {
+      enabled: jurisdictions.length > 0,
+      staleTime: config.staleTime !== undefined ? config.staleTime : Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)